### PR TITLE
[#RFW-45] Rosetta Android - Jorjin WG7833-B0 Wi-Fi and Bluetooth Integration

### PR DESCRIPTION
**Pull request by vinothraj-aricent**
updating the  wlan configurations for Jorjin WI-FI in make files

### CODE REVIEW

File: core/config.mk [hunk 1]
```diff
@@ -640,13 +640,13 @@ endif
 # Set up final options.
 # ###############################################################
 
-ifneq ($(COMMON_GLOBAL_CFLAGS)$(COMMON_GLOBAL_CPPFLAGS),)
```
Comment (leftbyte) [inline core/config.mk]: Why did this have to be commented out to add the wlan module?

**Comment by vinothraj-aricent** [comment]
Now the spelling mistake is corrected in the commit message.

File: core/config.mk (hunk 1 above)
Comment (leftbyte) [reply core/config.mk]: @vinothraj-aricent Can you please comment as to why we need to remove these flags?

Comment (Sangilikumar-Aricent) [reply core/config.mk]: We are defined  COMMON_GLOBAL_CFLAGS in Board file and using this flag for build the WLAN related library from different components. So we are removing this check.

Comment (leftbyte) [reply core/config.mk]: These global CFLAGS should really be defined here. If you have local wlan related flags, those should be defined with a different CFLAG variable.

Comment (Sangilikumar-Aricent) [reply core/config.mk]: Its TI supplied patch, we tested and used it for initial integration. We didn't optimize the TI supplied patch

Comment (priya-aricent) [reply core/config.mk]: @leftbyte - Can we track this optimization, similarly, as a JIRA issue?

Comment (mohan-aricent) [reply core/config.mk]: @leftbyte - Can we merge this change and track this review comment from another JIRA issue?

Comment (dbuckley000) [reply core/config.mk]: Dan - please merge as-is, we'll open another issue to track...  I can't build until this is merged.

Comment (leftbyte) [reply core/config.mk]: If you all are using supplied patches from TI, please note where you got those patches in the PR description.

**Pull request merged**